### PR TITLE
Fixed Button and Url

### PR DESCRIPTION
**Pull request by KumarUtsav1025**
Sir, In resolving the conflicts some file changes got reverted. I have fixed it in this PR.

### CODE REVIEW

File: pubspec.lock [hunk 1]
```diff
@@ -84,7 +84,7 @@ packages:
       name: collection
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "1.15.0"
+    version: "1.16.0"
   crypto:
```
Comment (TreavVasu) [inline pubspec.lock]: I don't see a new package being introduced in .yaml so i don't think there's a need for this file update. Resolve this and notify

Comment (KumarUtsav1025) [reply pubspec.lock]: Sir, whenever I am running pub get, the file is automatically getting updated.

Comment (TreavVasu) [reply pubspec.lock]: Then copy paste the original from repo and commit

Comment (KumarUtsav1025) [reply pubspec.lock]: Sir, I have made the required changes.

**Pull request merged**